running the kcb.sh now results in an un-handled execption requesting additional params for open or closed model user run rates

### Describe the bug

After this PR - https://github.com/keycloak/keycloak-benchmark/pull/43

I believe we introduced an issue with not handling the missing arguments to run the kcb.sh script.

If I follow the docs to run the default scenario on default host, it results in an exception requesting to provide one additional argument for the simulation to be started as Open or Closed model.

The same works just fine, once we provide the `--users-per-sec` for the OPEN workload model, or `--concurrent-users` for the CLOSED workload model along with the old arguments.

```
[kakella@kakella keycloak-benchmark]$ ./keycloak-benchmark-0.7-SNAPSHOT/bin/kcb.sh
13:33:45.656 [INFO ] a.e.s.Slf4jLogger - Slf4jLogger started
13:33:45.932 [ERROR] i.g.a.Gatling$ - Run crashed
java.lang.IllegalArgumentException: Unable to determine Gatling workload model. Specify either `--users-per-sec` for the OPEN workload model, or `--concurrent-users` for the CLOSED workload model.
	at org.keycloak.benchmark.Config.<clinit>(Config.java:77)
	... 16 common frames omitted
Wrapped by: java.lang.ExceptionInInitializerError: null
	at keycloak.scenario.KeycloakScenarioBuilder.userThinkPause(KeycloakScenarioBuilder.scala:123)
	at keycloak.scenario.KeycloakScenarioBuilder.openLoginPage(KeycloakScenarioBuilder.scala:147)
	at keycloak.scenario.KeycloakScenarioBuilder$.<init>(KeycloakScenarioBuilder.scala:42)
	at keycloak.scenario.KeycloakScenarioBuilder$.<clinit>(KeycloakScenarioBuilder.scala)
	at keycloak.scenario.KeycloakScenarioBuilder.clientCredentialsGrant(KeycloakScenarioBuilder.scala:271)
	at keycloak.scenario.authentication.ClientSecret.<init>(ClientSecret.scala:8)
	... 10 common frames omitted
Wrapped by: java.lang.reflect.InvocationTargetException: null
	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:490)
	at io.gatling.app.Runner.run0(Runner.scala:65)
	at io.gatling.app.Runner.run(Runner.scala:52)
	at io.gatling.app.Gatling$.start(Gatling.scala:80)
	at io.gatling.app.Gatling$.fromArgs(Gatling.scala:45)
	at io.gatling.app.Gatling$.main(Gatling.scala:37)
	at io.gatling.app.Gatling.main(Gatling.scala)
Exception in thread "main" java.lang.reflect.InvocationTargetException
	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:490)
	at io.gatling.app.Runner.run0(Runner.scala:65)
	at io.gatling.app.Runner.run(Runner.scala:52)
	at io.gatling.app.Gatling$.start(Gatling.scala:80)
	at io.gatling.app.Gatling$.fromArgs(Gatling.scala:45)
	at io.gatling.app.Gatling$.main(Gatling.scala:37)
	at io.gatling.app.Gatling.main(Gatling.scala)
Caused by: java.lang.ExceptionInInitializerError
	at keycloak.scenario.KeycloakScenarioBuilder.userThinkPause(KeycloakScenarioBuilder.scala:123)
	at keycloak.scenario.KeycloakScenarioBuilder.openLoginPage(KeycloakScenarioBuilder.scala:147)
	at keycloak.scenario.KeycloakScenarioBuilder$.<init>(KeycloakScenarioBuilder.scala:42)
	at keycloak.scenario.KeycloakScenarioBuilder$.<clinit>(KeycloakScenarioBuilder.scala)
	at keycloak.scenario.KeycloakScenarioBuilder.clientCredentialsGrant(KeycloakScenarioBuilder.scala:271)
	at keycloak.scenario.authentication.ClientSecret.<init>(ClientSecret.scala:8)
	... 10 more
Caused by: java.lang.IllegalArgumentException: Unable to determine Gatling workload model. Specify either `--users-per-sec` for the OPEN workload model, or `--concurrent-users` for the CLOSED workload model.
	at org.keycloak.benchmark.Config.<clinit>(Config.java:77)
	... 16 more
[kakella@kakella keycloak-benchmark]$ ./keycloak-benchmark-0.7-SNAPSHOT/bin/kcb.sh --scenario=keycloak.scenario.authentication.LoginUserPassword
13:34:18.094 [INFO ] a.e.s.Slf4jLogger - Slf4jLogger started
13:34:18.351 [ERROR] i.g.a.Gatling$ - Run crashed
java.lang.IllegalArgumentException: Unable to determine Gatling workload model. Specify either `--users-per-sec` for the OPEN workload model, or `--concurrent-users` for the CLOSED workload model.
	at org.keycloak.benchmark.Config.<clinit>(Config.java:77)
	... 16 common frames omitted
Wrapped by: java.lang.ExceptionInInitializerError: null
	at keycloak.scenario.KeycloakScenarioBuilder.userThinkPause(KeycloakScenarioBuilder.scala:123)
	at keycloak.scenario.KeycloakScenarioBuilder.openLoginPage(KeycloakScenarioBuilder.scala:147)
	at keycloak.scenario.KeycloakScenarioBuilder$.<init>(KeycloakScenarioBuilder.scala:42)
	at keycloak.scenario.KeycloakScenarioBuilder$.<clinit>(KeycloakScenarioBuilder.scala)
	at keycloak.scenario.KeycloakScenarioBuilder.openLoginPage(KeycloakScenarioBuilder.scala:131)
	at keycloak.scenario.authentication.LoginUserPassword.<init>(LoginUserPassword.scala:8)
	... 10 common frames omitted
Wrapped by: java.lang.reflect.InvocationTargetException: null
	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:490)
	at io.gatling.app.Runner.run0(Runner.scala:65)
	at io.gatling.app.Runner.run(Runner.scala:52)
	at io.gatling.app.Gatling$.start(Gatling.scala:80)
	at io.gatling.app.Gatling$.fromArgs(Gatling.scala:45)
	at io.gatling.app.Gatling$.main(Gatling.scala:37)
	at io.gatling.app.Gatling.main(Gatling.scala)
Exception in thread "main" java.lang.reflect.InvocationTargetException
	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:490)
	at io.gatling.app.Runner.run0(Runner.scala:65)
	at io.gatling.app.Runner.run(Runner.scala:52)
	at io.gatling.app.Gatling$.start(Gatling.scala:80)
	at io.gatling.app.Gatling$.fromArgs(Gatling.scala:45)
	at io.gatling.app.Gatling$.main(Gatling.scala:37)
	at io.gatling.app.Gatling.main(Gatling.scala)
Caused by: java.lang.ExceptionInInitializerError
	at keycloak.scenario.KeycloakScenarioBuilder.userThinkPause(KeycloakScenarioBuilder.scala:123)
	at keycloak.scenario.KeycloakScenarioBuilder.openLoginPage(KeycloakScenarioBuilder.scala:147)
	at keycloak.scenario.KeycloakScenarioBuilder$.<init>(KeycloakScenarioBuilder.scala:42)
	at keycloak.scenario.KeycloakScenarioBuilder$.<clinit>(KeycloakScenarioBuilder.scala)
	at keycloak.scenario.KeycloakScenarioBuilder.openLoginPage(KeycloakScenarioBuilder.scala:131)
	at keycloak.scenario.authentication.LoginUserPassword.<init>(LoginUserPassword.scala:8)
	... 10 more
Caused by: java.lang.IllegalArgumentException: Unable to determine Gatling workload model. Specify either `--users-per-sec` for the OPEN workload model, or `--concurrent-users` for the CLOSED workload model.
	at org.keycloak.benchmark.Config.<clinit>(Config.java:77)
	... 16 more
```



### Version

0.7-SNAPSHOT

### Expected behavior

The kcb.sh script should either let the user know there is a new argument needed or make sure there are default values picked up for the open or closed model similar to previous code

### Actual behavior

If we dont provide the argument `--users-per-sec` for the OPEN workload model, or `--concurrent-users` for the CLOSED workload model, the script fails with an unhandled exception.

### How to Reproduce?

run the below command, as specified in the README.md

`kcb.sh`

or 

`kcb.sh --scenario=keycloak.scenario.authentication.LoginUserPassword`

### Anything else?

#### Proposed fix:
We can add additional exception handling in the kcb.sh to ensure to pass the needed arguments, as well as update the README.md with the new parameter. I am working on a fix for the same, but wanted to call out, if someone from the community is also planning on using the latest kcb release binaries.

--- a/benchmark/src/main/java/org/keycloak/benchmark/Config.java
+++ b/benchmark/src/main/java/org/keycloak/benchmark/Config.java
@@ -77,16 +77,17 @@ public class Config {
     public static final WorkloadModel workloadModel;
     
     static {
-        usersPerSec = Double.valueOf(System.getProperty("users-per-sec", "0"));
+        double usersPerSecTmp = Double.valueOf(System.getProperty("users-per-sec", "0"));
         concurrentUsers = Integer.valueOf(System.getProperty("concurrent-users", "0"));
-        if (usersPerSec > 0 && concurrentUsers == 0) {
+        if (usersPerSecTmp > 0 && concurrentUsers == 0) {
             workloadModel = WorkloadModel.OPEN;
-        } else if (concurrentUsers > 0 && usersPerSec == 0) {
+        } else if (concurrentUsers > 0 && usersPerSecTmp == 0) {
             workloadModel = WorkloadModel.CLOSED;
         } else {
-            throw new IllegalArgumentException("Unable to determine Gatling workload model. "
-                    + "Specify either `--users-per-sec` for the OPEN workload model, or `--concurrent-users` for the CLOSED workload model.");
+            workloadModel = WorkloadModel.OPEN; // default, like "--users-per-sec=1" was specified
+            usersPerSecTmp=1.0;
         }
+        usersPerSec = usersPerSecTmp;
     }
     
     /**

